Code/GraphMol/SmilesParse/SmilesWrite.h

```diff
@@ -30,8 +31,12 @@ struct RDKIT_SMILESPARSE_EXPORT SmilesWriteParams {
                             is not canonical and that this will thrown an
                             exception if the molecule cannot be kekulized. */
   bool canonical = true; /**< generate canonical SMILES */
-  bool allBondsExplicit = false; /**< include symbols for all bonds */
-  bool allHsExplicit = false;    /**< provide hydrogen counts for every atom */
+  bool rigorousEnhancedStereo = false; /**< if true, use a more rigorous
```

Comment:
Seems like this should at least default to true.
It's worth considering having it not even be an option.

Reply:
Please move this, and the other new option, to the end of the struct so that it's easier to adapt code which uses `{}` initialization of the struct

Reply:
I'm ok with the default being true, but that would change the behavior of existing applications that use it.  Your call.

I will move it to the end of the list as suggested.

Reply:
> I'm ok with the default being true, but that would change the behavior of existing applications that use it. Your call.

My thinking here: the default behavior of MolToSmiles() is to produce canonical SMILES. The changes in this PR are about doing that (at least that's how I understand them).

Reply:
OK I will change the default to true.